packages/core/src/charts/combo.ts

```diff
@@ -0,0 +1,99 @@
+// Internal Imports
+import { AxisChart } from "../axis-chart";
+import * as Configuration from "../configuration";
+import {
+	ChartConfig,
+	ComboChartOptions
+} from "../interfaces/index";
+import { Tools } from "../tools";
+import { ChartModel } from "../model";
+
+// Components
+import {
+	Grid,
+	GroupedBar,
+	SimpleBar,
+	StackedBar,
+	Line,
+	TwoDimensionalAxes,
+	ZeroLine,
+	Scatter,
+	Ruler,
+	// the imports below are needed because of typescript bug (error TS4029)
+	Legend,
+	LayoutComponent
+} from "../components/index";
+
+const graphComponentsMap = {
+	"line": [Line, Scatter],
+	"simple-bar": [SimpleBar, ZeroLine],
+	"grouped-bar": [GroupedBar, ZeroLine],
+	"stacked-bar": [StackedBar]
+};
+
+
+export class ComboChart extends AxisChart {
+	model = new ChartModel(this.services);
+
+	constructor(holder: Element, chartConfigs: ChartConfig<ComboChartOptions>) {
+		super(holder, chartConfigs);
+
+		// Warn user if no chartTypes defined
+		if (!chartConfigs.options.chartTypes) {
+			console.warn("No chartTypes defined for the Combo Chart!");
+		}
+
+		// Merge the default options for this chart
+		// With the user provided options
+		const chartOptions = Tools.mergeDefaultChartOptions(
+			Configuration.options.comboChart,
+			chartConfigs.options
+		);
+
+		// Merge multiple graphs default options
+		const { chartTypes } = chartOptions;
+		const graphs = Object.keys(chartTypes);
+		const graphsDefaultOptions = graphs.reduce((options, g) => Tools.merge(options, Configuration.options[`${Tools.camelCase(g)}Chart`]), {});
+
+		// Merge default, user provided and graphs default options
+		this.model.setOptions(
+			Tools.mergeDefaultChartOptions(
+				graphsDefaultOptions,
+				chartOptions
+			)
+		);
+
+		// If the stacked-bar chart is included, stack its data groups
+		if (graphs.includes("stacked-bar")) {
+			this.model.setStackedGroups(chartTypes["stacked-bar"]);
+		}
+
+		// Initialize data, services, components etc.
+		this.init(holder, chartConfigs);
+	}
+
+	getGraphComponents(graph: string) {
```

Comment:
we need a better more descriptive name here

Reply:
there was a `getGraphsComponents` and `getGraphComponents` but i changed it so theres just this one so that should be more clear. Do you still want the naming changed more or was that the issue?